code/modules/reagents/machinery/mortar.dm

```diff
@@ -70,9 +24,11 @@
 	if (crushable)
 		to_chat(user, "<span class ='warning'>There's already something inside!</span>")
 		return 1
-	if (!is_type_in_list(O, blend_items) && !is_type_in_list(O, juice_items))
-		to_chat(user, "<span class ='warning'>You can't grind that!</span>")
-		return ..()
+
+	//There used to be a check here to only allow certain whitelisted things to be put inside the mortar. Now we just check size. God help us.
```

Comment:
At the end of the day, there's just no reason to make it so that you can't hide tiny items inside grinders. There's a gameplay reason why they shouldn't be destroyed by the grinder, but there's no reason to stop them from going in entirely.

Now, if you can think of an item that needs to be grinded that is W_CLASS_MEDIUM or larger you might be more convincing. Otherwise this is a silly thing to be so autistic over.